kickstarter/android-oss
# NTV-190: Help – Contact button not working

# 📲 What

Help – Contact button not working

# 🛠 How
![image](https://user-images.githubusercontent.com/1075310/133828464-f73e67e1-fbcf-40c2-8a82-ab377ab44a60.png)

# 👀 See
https://user-images.githubusercontent.com/1075310/133828416-4a222a23-8e26-43bb-a167-2ff5f144c717.mp4

# 📋 QA

1. Make sure you are logged in
2. Go to Settings > Help
3. Tap Contact
4. Nothing happens

# Story 📖

https://kickstarter.atlassian.net/browse/NTV-190
